libsplinter/src/biome/migrations/mod.rs

```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Provides sql migration scripts and methods for executing
```

Comment:
This commit says "for biome", but it should say "for biome migrations"

Reply:
Updated commit message